### ISSUE TITLE
Firefox: Cannot access dead object

### ISSUE THREAD

**Issue by icio** [unravelin/ravelinjs#105]
We're getting errors in v1.1.2 and v1.2, from Firefox user agents (e.g. Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:89.0) Gecko/20100101 Firefox/89.0) of the form:

```
 TypeError: can't access dead object
    at s (ravelin-1.1.2-core-track.min.js:2:3326)
    at Ravelin</o.prototype.resize (ravelin-1.1.2-core-track.min.js:2:16741)
    at Ravelin</n.prototype.bind/</< (ravelin-1.1.2-core-track.min.js:2:13726)
    at Ravelin</n.prototype.sniffError (ravelin-1.1.2-core-track.min.js:2:13423)
    at Ravelin</n.prototype.bind/< (ravelin-1.1.2-core-track.min.js:2:13694)
    at c/</t< (ravelin-1.1.2-core-track.min.js:2:13123)
    at init (site.js:1:1)
```

Googling around suggests that it's something to do with an extension holding onto a reference for too long, but I've tried all of the top firefox extensions and not been able to reproduce. The exception is always thrown within Track.resize, so perhaps our event handler is fired when the browser is closing?

**Comment by icio** [comment]
In the stack trace, function s of the minified script corresponds to:

```
function s(){return{w:window.outerWidth||window.innerWidth||document.documentElement.clientWidth||document.body.clientWidth,h:window.outerHeight||window.innerHeight||document.documentElement.clientHeight||document.body.clientHeight}}
```

which is our: https://github.com/unravelin/ravelinjs/blob/790bfffb12aa9450d49fbfabb6e7ad687592e5e3/lib/track.js#L172-L177

So I guess lets put a try-catch around it? If I'm correct in understanding that this "resize" event is actually the browser closing then I think we're fine to ignore it.

**Comment by icio** [comment]
Fixed in [v1.3.0](https://github.com/unravelin/ravelinjs/releases/tag/v1.3.0)
